src/components/watermark/watermark.tsx

```diff
@@ -90,7 +104,39 @@ class Watermark extends Component<any, any> {
       }
     }
   }
-  
+
+  private _loadImageDimension(): void {
+    const img = new Image();
+    img.src = this.props.config.img;
+    img.onload = () => {
+      this._imgWidth = img.naturalWidth;
```

Comment:
@this._imgWidth / height stores the ORIGINAL values, so please rename them in order to distinguish them from the values in the state